packages/hop-node/src/watchers/BondWithdrawalWatcher.ts

```diff
@@ -61,7 +64,25 @@ class BondWithdrawalWatcher extends BaseWatcher {
     }
 
     const promises: Promise<any>[] = []
-    for (const { transferId } of dbTransfers) {
+    for (const dbTransfer of dbTransfers) {
+      const {
+        transferId,
+        destinationChainId,
+        amount,
+        withdrawalBondTxError
+      } = dbTransfer
+
+      const sourceChain = this.bridge.chainSlug
+      const destinationChain = this.chainIdToSlug(destinationChainId)
+      const lastAvailableCredit = this.lastAvailableCredit?.[sourceChain]?.[destinationChain]
+      if (
+        lastAvailableCredit &&
```

Comment:
The first conditional can be omitted and the second can use `?` to simply things

```js
      if (
        lastAvailableCredit?.lt(amount) &&
        withdrawalBondTxError === TxError.NotEnoughLiquidity
      ) {
    
```